backend/app/services/adapters/team_kinds.py

```
@@ -2142,5 +2142,172 @@ def get_team_skills(
+            )
+            bot_ref = leader_member.get("botRef", {})
+
+            # Find the original bot Kind record
+            original_bot = (
+                db.query(Kind)
+                .filter(
+                    Kind.kind == "Bot",
+                    Kind.name == bot_ref.get("name"),
+                    Kind.namespace == bot_ref.get("namespace", original.namespace),
+                    Kind.user_id == original.user_id,
+                    Kind.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)